src/pages/api/index.tsx

```diff
@@ -0,0 +1,266 @@
+                                    </ClientCodeWrapper>
+                                </SectionWrapper>
+                            ),
+                        },
+                        {
+                            title: <TabTitleWrapper>
+                                <ThemedImage
+                                    height={16}
+                                    width={16}
+                                    sources={{ dark: useBaseUrl('/img/python-40x40.svg'), light: useBaseUrl('/img/python-40x40.svg') }}
+                                />
+                                Python Client
+                            </TabTitleWrapper>,
+                            content: (
+                                <SectionWrapper
+                                    heading="Python API client"
+                                    description={<div className="Description">
+                                        The official library to interact with Apify API from a Python applications.
+                                        <GitHubButton href="https://github.com/apify/apify-client-python" data-color-scheme="no-preference: light; light: light; dark: dark;" data-size="large" data-show-count="true">Star</GitHubButton>
+                                        <div className="DescriptionLinks">
+                                            <Button to='https://docs.apify.com/api/client/python/docs'>Get started</Button>
+                                            <ActionLink to='https://docs.apify.com/api/client/python'>Python client reference</ActionLink>
+                                        </div>
+                                    </div>}
+                                >
+                                    <ClientCodeWrapper>
+                                        <CodeBlock content='pip install apify-client' language="bash"/>
+                                        <CodeBlock
+                                            className="codeBlock"
+                                            language='python'
+                                            hideLineNumbers
+                                            content={`
+from apify_client import ApifyClient
+
+apify_client = ApifyClient('MY-APIFY-TOKEN')
+
+# Start an actor and wait for it to finish
+actor_call = apify_client.actor('john-doe/my-cool-actor').call()
+
+# Fetch results from the actor run's default dataset
+dataset_items = apify_client.dataset(actor_call['defaultDatasetId']).list_items().items
+                                `}/></ClientCodeWrapper>
+                                </SectionWrapper>
+                            ),
+                        },
+                    ]} />
+                </Section>
+                <Section heading="Related articles">
+                    <RelatedArticlesWrapper>
+                        <a href="https://blog.apify.com/web-scraping-with-client-side-vanilla-javascript/">
+                            <BlogArticle
```

Comment:
Wouldn't be better to define the IDs of the blog post and fetch them dynamically so the title and URL change if the blog post changes?

Reply:
Yeah, fair. Let's not complicate it